src/app.css

```
@@ -253,6 +256,90 @@ body {
+@media (max-width: 768px) {
+  .page-header-content {
+    flex-direction: column;
+    gap: var(--space-4);
+  }
+}
+
+@media (max-width: 480px) {
+  .page-actions {
+    width: 100%;
+  }
+  .page-actions :global(button) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)